vncviewer/vncviewer.cxx

```diff
@@ -587,9 +587,23 @@ createTunnel(const char *gatewayHost, const char *remoteHost,
   const char *cmd = getenv("VNC_VIA_CMD");
   char *cmd2, *percent;
   char lport[10], rport[10];
+  char *tmps, *tmps2, *tmphead;
+
+  tmps = tmphead = strdup (gatewayHost);
+
+  /* strip leading whitespace from gatewayHost */
+  while (*tmps != '\0' && isspace(*tmps)) tmps++;
+
+  /* strip trailing whitespace from gatewayHost */
+  tmps2 = tmps;
+  while (*tmps2 != '\0' && !isspace(*tmps2)) tmps2++;
+  if (isspace (*tmps2)) *tmps2 = '\0';
+  vlog.info(_("Establish SSH tunnel to gatewayHost \"%s\"\n"), tmps);
+
```

Comment:
These are nice improvements, but unrelated to adding the GUI. So please have this as a separate commit.

Reply:
Yes, that's precisely right. :)